resources/event-bus/charts/publish-knative/templates/deployment.yaml

```diff
@@ -0,0 +1,20 @@
+# dummy pod   
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
+kind: Deployment
```

Comment:
Looks like not needed?

Reply:
It's deliberately added like that as a place holder for the coming real implementation in the coming pull requests.

Reply:
ok